closed/src/java.base/share/native/libjncrypto/NativeCrypto.c

```diff
@@ -38,10 +39,13 @@
 #include "jdk_crypto_jniprovider_NativeCrypto.h"
 #include "NativeCrypto_md.h"
 
-#define OPENSSL_VERSION_1_0 "OpenSSL 1.0."
-#define OPENSSL_VERSION_1_1 "OpenSSL 1.1."
+#define OPENSSL_VERSION_CODE(major, minor) (((major) << 24) | ((minor) << 20))
+#define OPENSSL_VERSION_1_0_0 OPENSSL_VERSION_CODE(1, 0)
```

Comment:
Please leave a blank line before the definition of `OPENSSL_VERSION_1_0_0`.

Reply:
Edited.